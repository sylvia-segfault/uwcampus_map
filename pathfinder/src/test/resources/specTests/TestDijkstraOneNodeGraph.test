# Find a path with Dijkstra's algorithm on
# a graph that contains one node

CreateGraph graph
AddNode graph A
FindPath graph A B