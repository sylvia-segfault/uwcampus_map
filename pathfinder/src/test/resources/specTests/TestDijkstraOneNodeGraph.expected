# Find a path with Dijkstra's algorithm on
# a graph that contains one node

created graph graph
added node A to graph
unknown node B